backend/api/Profiles/BaseProfile.cs

```diff
@@ -17,6 +17,8 @@ public BaseProfile()
             CreateMap<BaseModel, Entity.BaseEntity>()
                 .IncludeAllDerived()
                 .ForMember(dest => dest.RowVersion, opt => opt.Ignore())
+                .ForMember(dest => dest.CreatedById, opt => opt.Ignore())
```

Comment:
Do we actually want to pass this information all the time?  I didn't include them originally because I didn't think they had a lot value in the majority of cases.

Reply:
The profile is ignoring this information, meaning it will NOT be passed.
I've added this and some of the other Automapper in an effort to get our mapping configurations passing validation: 
 mapperConfig.AssertConfigurationIsValid();

Reply:
Cheer for my reading code quickly abilities :(

Reply:
If I'm reading correctly, we are ignoring "Created/Updated By" fields when we go from JSON models to DB entities, right? If that's the case I think this code is useful as I wouldn't want to let external actors be able to "override" audit data via a JSON PUT/PUT

Can someone tell me if I'm wrong here?